doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2472 @@
+
+If you start with this array:
+
+::
+
+  array = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can use ``flatten`` to flatten your array into a 1D array.
+
+**Input:**
+
+::
+
+  array.flatten()
+
+**Output:**
+
+::
+
+  array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12])
+
+When you use ``flatten``, changes to your new array won't change the parent array.
+
+For example:
+
+**Input:**
+
+::
+
+  a1 = array.flatten()  
+  a1[0] = 100
+  print('Original array: ')
+  print(array)
+  print('New array: ')
+  print(a1)
+
+**Output:**
+
+::
+
+  Original array: 
+  [[ 1  2  3  4]
+   [ 5  6  7  8]
+   [ 9 10 11 12]]
+  New array: 
+  [100   2   3   4   5   6   7   8   9  10  11  12]
+
+
+But when you use ``ravel``, the changes you make to the new array will affect the parent array.
+
+For example:
+
+**Input:**
+
+::
+
+  a2 = array.ravel()  
+  a2[0] = 101 
+  print('Original array: ')
+  print(array)
+  print('New array: ')
+  print(a2)
+
+**Output:**
+
+::
+
+  Original array: 
+  [[101   2   3   4]
+   [  5   6   7   8]
+   [  9  10  11  12]]
+  New array: 
+  [101   2   3   4   5   6   7   8   9  10  11  12]
+
+`Read more about flatten here <https://numpy.org/devdocs/reference/generated/numpy.ndarray.flatten.html>`_ and `ravel here <https://numpy.org/devdocs/reference/generated/numpy.ravel.html#numpy.ravel>`_.
+
+
+How to access the docstring for more information
+------------------------------------------------
+
+::
+
+  help()
+  ?
+  ??
+
+-----
+
+When it comes to the data science ecosystem, Python and NumPy are built with the user in mind. One of the best examples of this is the built-in access to documentation. Every object contains the reference to a string, which is known as the **docstring**. In most cases, this docstring contains a quick and concise summary of the object and how to use it. Python has a built-in ``help()`` function that can help you access this information. This means that nearly any time you need more information, you can use ``help()`` to quickly find the information that you need.
+
+For example,
+
+::
+
+  help(max)
+
+Will return
+
+::
+
+  Help on built-in function max in module builtins:
+
+  max(...)
+      max(iterable, *[, default=obj, key=func]) -> value
+      max(arg1, arg2, *args, *[, key=func]) -> value
+      
+      With a single iterable argument, return its biggest item. The
+      default keyword-only argument specifies an object to return if
+      the provided iterable is empty.
+      With two or more arguments, return the largest argument.
+
+Because access to additional information is so useful, IPython uses the ``?`` character as a shorthand for accessing this documentation along with other relevant information. IPython is a command shell for interactive computing in multiple languages.`You can find more information about IPython here <https://ipython.org/>`_. 
```

Comment:
For the sake of code review comments, I'd recommend one sentence per line rather than one paragraph per line.

What I actually wanted to comment here is I think this will fix your rendering, maybe:
```suggestion
Because access to additional information is so useful, IPython uses the ``?`` character as a shorthand for accessing this documentation along with other relevant information. IPython is a command shell for interactive computing in multiple languages. `You can find more information about IPython here <https://ipython.org/>`_. 
```